docs/payments-core/com.stripe.android.exception/-card-exception/index.html

```diff
@@ -62,7 +62,7 @@
   <div class="cover ">
     <h1 class="cover"><span>Card</span><wbr><span><span>Exception</span></span></h1>
-    <div class="platform-hinted " data-platform-hinted="data-platform-hinted"><div class="content sourceset-dependent-content" data-active="" data-togglable=":payments-core:dokkaHtmlPartial/release"><div class="symbol monospace"><span class="token keyword">class </span><a href="index.html">CardException</a><span class="token punctuation">(</span><span class="parameters "><span class="parameter ">stripeError<span class="token operator">: </span><span data-unresolved-link="com.stripe.android.core/StripeError///PointingToDeclaration/">StripeError</span><span class="token punctuation">, </span></span><span class="parameter ">requestId<span class="token operator">: </span><a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin-stdlib/kotlin/-string/index.html">String</a><span class="token operator">?</span><span class="token operator"> = </span>null</span></span><span class="token punctuation">)</span> : <span data-unresolved-link="com.stripe.android.core.exception/StripeException///PointingToDeclaration/">StripeException</span></div><p class="paragraph">An <a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin-stdlib/kotlin/-exception/index.html">Exception</a> indicating that there is a problem with a Card used for a request. Card errors are the most common type of error you should expect to handle. They result when the user enters a card that can't be charged for some reason.</p></div></div>
+    <div class="platform-hinted " data-platform-hinted="data-platform-hinted"><div class="content sourceset-dependent-content" data-active="" data-togglable=":payments-core:dokkaHtmlPartial/release"><div class="symbol monospace"><span class="token keyword">class </span><a href="index.html">CardException</a><span class="token punctuation">(</span><span class="parameters "><span class="parameter ">stripeError<span class="token operator">: </span><span data-unresolved-link="com.stripe.android.core/StripeError///PointingToDeclaration/">StripeError</span><span class="token punctuation">, </span></span><span class="parameter ">requestId<span class="token operator">: </span><a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin-stdlib/kotlin/-string/index.html">String</a><span class="token operator">?</span><span class="token operator"> = </span>null</span></span><span class="token punctuation">)</span> : <span data-unresolved-link="com.stripe.android.core.exception/StripeException///PointingToDeclaration/">StripeException</span></div><p class="paragraph">An <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Exception.html">Exception</a> indicating that there is a problem with a Card used for a request. Card errors are the most common type of error you should expect to handle. They result when the user enters a card that can't be charged for some reason.</p></div></div>
```

Comment:
Our PII checker identified `https://docs.oracle.com` as PII 🤦 

I don't expect this to be a persistent problem, but if it is, we can add it to the allow list.